osu.Game/Screens/Play/DimmableStoryboard.cs

```diff
@@ -24,6 +27,8 @@ public DimmableStoryboard(Storyboard storyboard)
         [BackgroundDependencyLoader]
         private void load()
         {
+            Add(OverlayLayerContainer = new Container());
```

Comment:
I think you should auto-size this to the overlay layer?

Reply:
Checked this one - setting auto-size on this container has no impact on anything as it has 0x0 size regardless (draw visualiser shows that the only child of this container is a 0x0 `ProxyDrawable`). Sizing of the actually-drawn layer is dealt with by proxying, so I'm inclined to leave this as is.